.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+Resolves PROJ-XYZ
```

Comment:
For optimum effect, make sure to [configure autolink references](https://github.com/ithaka/aiw-ui/settings/key_links) for this repo 😄